### ISSUE TITLE
Pull request for new functionality

### ISSUE THREAD

**Issue by micio86dev** [LucianoBestia/chext_fill_form_from_json#3]
Hi I developed a read from Url functionality but I can't open a new pull request because you have to open another branch (develop or other name). I can't request a pull on master branch.

Tnx

**Comment by LucianoBestia** [comment]
Great. I published a new branch. Add your pull request there. Do you have also a test url, so I can test it?

**Comment by micio86dev** [comment]
Hi, I have a test URL buy you can use it only once. But the mod can read
from any URL like: https://tools.learningcontainer.com/sample-json.json

Il ven 17 dic 2021, 07:25 bestia.dev ***@***.***> ha scritto:
